gradoop-flink/src/main/java/org/gradoop/flink/model/api/epgm/LogicalGraphOperators.java

```diff
@@ -559,6 +559,15 @@ GraphCollection groupEdgesByRollUp(
     List<String> vertexGroupingKeys, List<AggregateFunction> vertexAggregateFunctions,
     List<String> edgeGroupingKeys, List<AggregateFunction> edgeAggregateFunctions);
 
+  /**
+   * Verifies this graph, removing dangling edges, i.e. edges pointing to or from
+   * a vertex not contained in this graph.<br>
+   * This operator may be applied after an operator that did not check for dangling edges.
```

Comment:
`This operator can be applied after an operator that has not checked the graphs validity.`
Please also add: `The graph head of this logical graph remains unchanged.`